modules/ibmz-configure-nbde-with-static-ip.adoc

```diff
@@ -120,35 +120,38 @@ ifndef::ibm-z-kvm[]
 ----
 rd.neednet=1 \
 console=ttysclp0 \
-coreos.inst.install_dev=/dev/dasda \ <1>
+coreos.inst.install_dev=/dev/dasda \// <1>
 ignition.firstboot ignition.platform.id=metal \
-coreos.live.rootfs_url=http://10.19.17.25/redhat/ocp/rhcos-413.86.202302201445-0/rhcos-413.86.202302201445-0-live-rootfs.s390x.img \
-coreos.inst.ignition_url=http://bastion.ocp-cluster1.example.com:8080/ignition/master.ign \
```

Comment:
@madhu-pillai If I remember correctly I worked with you on first setting up NBDE docs. Does this have to be the master.ign or can it be `bootstrap.ign`, `master.ign`, or `worker.ign`?

Reply:
We had given an example referring master.ign for NBDE.  It can be master or worker depends on which node it is intent .  We do not do disk encryption on bootstrap.